docs/source/releases.rst

```diff
@@ -2,6 +2,105 @@ Release History
+
+Library Features
+----------------
+- Added the ability to add custom units for convert_unit with a unit test
+- Add the capability for `AvgSignal` to reset itself on trigger,
+  then wait a duration before marking the trigger as complete.
+  This lets you use `AvgSignal` in a bluesky plan as a way to
+  accumulate the time average of a fast-updating signal at each
+  scan point. To enable this, provide a ``duration`` argument.
+
+Device Features
+---------------
+- `KBOMirrorHE` gets 2 new RTD readouts for RTDs installed on the mirror inside vaccum.
+- Add `calibrated_uj` and `manual_in_progress` components to `pcdsdevices.pim.PPMPowerMeter`
+- Renamed manual-collections components to have manual in their name to distinguish them from auto-background collection
+- Add the ``desc``, ``last_voltage_set``, and ``is_trip`` component signals to
+  `MPODApalisChannel`. These have been helpful during operations at TMO.
+  ``last_voltage_set`` will also get a ``voltage_setpoint`` alias, which is the
+  original name as used in TMO's scripts.
+- Add proper control limits to `MPODApalisChannel.voltage` and `MPODApalisChannel.current`.
+  This will give useful errors when someone tries to put values outside of the
+  channel's supported range.
+- Adjust `pcdsdevices.tmo_ip1.CalibrationAxis` from 5 -> 7 -> 8 states
+- `pcdsdevices.spectrometer.Mono` now has a `MonoGratingStates` component (`grating_states`)
+- `pcdsdevices.mirror.XOffsetMirrorSwitch` now has a `TwinCATMirrorStripe` component (`coating`)
+- Add `FDQ` to `pcdsdevices.spectrometer.TMOSpectrometer` (`flow_meter`)
+- Recover sp1k4 attenuator targets number from 3 to 5
+- Add one additional (SiN) target into TM1K4 (8 targets, 9 states with OUT)
+- `pcdsdevices.mirror.FFMirror` now has a `TwinCATMirrorStripe` component (`coating`)
+  - `FFMirrorZ` inherits these and sets them `None`
+- `pcdsdevices.slits.ExitSlits` now has an `FDQ` component (`flow_meter`)
+- Allow init-time configuration of phase shifter inversion and setpoint limits
+  for `LaserTiming` and `LCLS2LaserTiming` devices.
+
+New Devices
+-----------
+- `HxrDiffractometer` for the Beckhoff-based HXR diffractometer.
+  This controls the diffractometer in XPP with prefix ``"HXR:GON:MMS"``.
+- `MonoGratingStates`: SP1K1 Mono Grating States Axis G_H with PMPS.
+- `MirrorStripe2D2P`: Mirror Stripe with 2D Coating states, 2 positions and PMPS.
+- `PhotonCollimatorFDQ`: Photon Collimator with Cooling Meter Readback
+- `BeckhoffPneumaticFDQ`: Beckhoff Pneumatics with a flow meter for cooling readback
+- `VCN_OpenLoop`: similar to VCN w/ the removal of 'open' and 'position_readback'
+  commands. The 'state' member variable has been renamed to 'control_mode' and
+  the associated doc string was been updated.
+- `Lakeshore336`: support for Lakeshore 336 temperature controller
+
+Bugfixes
+--------
+- Made responsivity component input only like the pytmc pragma so the connection does not fail when looking for the non-RBV pv
```

Comment:
This line sticks out to me since it doesn't have any context in the big release notes.
Maybe this is:
```suggestion
- Made `PPMPowerMeter`'s ``responsivity`` component input-only, like the pytmc pragma, so the connection does not fail when looking for a non-existant non-RBV pv.
```